core/silkworm/common/util.hpp

```diff
@@ -82,4 +83,51 @@ size_t prefix_length(ByteView a, ByteView b);
 
 inline ethash::hash256 keccak256(ByteView view) { return ethash::keccak256(view.data(), view.size()); }
 
+//! \brief Create an intx::uint256 from a string supporting both fixed decimal and scientific notation
+template <typename Int>
+inline constexpr Int from_string_sci(const char* str) {
```

Comment:
Better pass as `std::string_view` for generality

Reply:
Right, done in commit https://github.com/torquem-ch/silkworm/pull/783/commits/040e804b018850386886a780545acd61522d4295